tests/online_migration/sle12_online_migration/post_migration.pm

```diff
@@ -14,13 +14,16 @@ use base "installbasetest";
 use strict;
 use testapi;
 use utils;
+use qam qw/add_test_repositories remove_test_repositories/;
 
 sub run {
     select_console 'root-console';
 
     # print repos to screen and serial console after online migration
     zypper_call('lr -u');
 
+    add_maintenance_repos() if(get_var('MAINT_TEST_REPO',''));
```

Comment:
why are you supplying a default when you just want to check for existance? Also, please add a space after `if`